dev-infra/commit-message/validate.ts

```diff
@@ -20,7 +20,7 @@ const SQUASH_PREFIX_RE = /^squash! /i;
 const REVERT_PREFIX_RE = /^revert:? /i;
 const TYPE_SCOPE_RE = /^(\w+)(?:\(([^)]+)\))?\:\s(.+)$/;
 const COMMIT_HEADER_RE = /^(.*)/i;
-const COMMIT_BODY_RE = /^.*\n\n(.*)/i;
+const COMMIT_BODY_RE = /^.*\n\n((.*|\n)*)/i;
```

Comment:
Won't matter much here, but this can lead to [catastrophic backtracking][1] in non-matching commit messages.
A better RegExp would be:

```suggestion
const COMMIT_BODY_RE = /^.*\n\n([\s\S]*)$/;
```

...or:

```suggestion
const COMMIT_BODY_RE = /^.*\n\n([^.]*)$/i;
```

##
In modern JS, one can also use the [dotAll (s) flag][2] to make `.` match newlines as well, but here it would make the first `.*` also match newlines, so you would need something like: `/^[\n]*\n\n(.*)$/s`

[1]: https://www.rexegg.com/regex-explosive-quantifiers.html
[2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/dotAll